### PR TITLE
 Added 201-correlation-logic-app-using-servicebus template

### PR DESCRIPTION
**Pull request by EldertGrootenboer**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### CODE REVIEW

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,740 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "Service Bus Namespace (should be globally unique!)": {
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: see BP guide for param/var naming conventions

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,740 @@
...
+            "defaultValue": "Router20180407",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: remove this or use uniqueString()

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,740 @@
...
+            "type": "String",
+            "metadata": {
+                "description":"The name of the connection to Service Bus created for Logic Apps."
+            }
+        }
+    },
+    "variables": {},
+    "resources": [
+        {
+            "comments": "Generalized from resource: '/subscriptions/fdf3a3a3-c8f5-472f-8367-6a9a4a6c11a9/resourceGroups/CorrelatingMessages/providers/Microsoft.Logic/workflows/BackendLogicApp'.",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: remove comments

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,740 @@
...
+            "type": "Microsoft.Logic/workflows",
+            "name": "[parameters('Backend Logic App name')]",
+            "apiVersion": "2017-07-01",
+            "location": "westeurope",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: use a parameter or resourceGroup().location

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 5]
```diff
@@ -0,0 +1,740 @@
...
+                    },
+                    "outputs": {}
+                },
+                "parameters": {
+                    "$connections": {
+                        "value": {
+                            "servicebus": {
+                                "connectionId": "[resourceId('Microsoft.Web/connections', parameters('Service Bus connection name'))]",
+                                "connectionName": "servicebus",
+                                "id": "/subscriptions/fdf3a3a3-c8f5-472f-8367-6a9a4a6c11a9/providers/Microsoft.Web/locations/westeurope/managedApis/servicebus"
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: use a valid resourceId

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 6]
```diff
@@ -0,0 +1,740 @@
...
+                        }
+                    }
+                }
+            },
+            "dependsOn": [
+                "[resourceId('Microsoft.Web/connections', parameters('Service Bus connection name'))]"
+            ]
+        },
+        {
+            "comments": "Generalized from resource: '/subscriptions/fdf3a3a3-c8f5-472f-8367-6a9a4a6c11a9/resourceGroups/CorrelatingMessages/providers/Microsoft.Logic/workflows/ClientLogicApp'.",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: remove comments

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 7]
```diff
@@ -0,0 +1,740 @@
...
+            "type": "Microsoft.Logic/workflows",
+            "name": "[parameters('Client Logic App name')]",
+            "apiVersion": "2017-07-01",
+            "location": "westeurope",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: use a parameter or resourceGroup().location

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 8]
```diff
@@ -0,0 +1,740 @@
...
+                    },
+                    "outputs": {}
+                },
+                "parameters": {
+                    "$connections": {
+                        "value": {
+                            "servicebus": {
+                                "connectionId": "[resourceId('Microsoft.Web/connections', parameters('Service Bus connection name'))]",
+                                "connectionName": "servicebus",
+                                "id": "/subscriptions/fdf3a3a3-c8f5-472f-8367-6a9a4a6c11a9/providers/Microsoft.Web/locations/westeurope/managedApis/servicebus"
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: use a valid resourceId

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 9]
```diff
@@ -0,0 +1,740 @@
...
+                            }
+                        }
+                    }
+                }
+            },
+            "dependsOn": [
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: move this above properties

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 10]
```diff
@@ -0,0 +1,740 @@
...
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: move this above properties

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 11]
```diff
@@ -0,0 +1,740 @@
...
+                "[resourceId('Microsoft.Web/connections', parameters('Service Bus connection name'))]"
+            ]
+        },
+        {
+            "comments": "Generalized from resource: '/subscriptions/fdf3a3a3-c8f5-472f-8367-6a9a4a6c11a9/resourceGroups/CorrelatingMessages/providers/Microsoft.Logic/workflows/TransformationLogicApp'.",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: remove comments

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 12]
```diff
@@ -0,0 +1,740 @@
...
+            "type": "Microsoft.Logic/workflows",
+            "name": "[parameters('Transformation Logic App name')]",
+            "apiVersion": "2017-07-01",
+            "location": "westeurope",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: use a parameter or resourceGroup().location

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 13]
```diff
@@ -0,0 +1,740 @@
...
+                    },
+                    "outputs": {}
+                },
+                "parameters": {
+                    "$connections": {
+                        "value": {
+                            "servicebus": {
+                                "connectionId": "[resourceId('Microsoft.Web/connections', parameters('Service Bus connection name'))]",
+                                "connectionName": "servicebus",
+                                "id": "/subscriptions/fdf3a3a3-c8f5-472f-8367-6a9a4a6c11a9/providers/Microsoft.Web/locations/westeurope/managedApis/servicebus"
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: use a valid resourceId

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 14]
```diff
@@ -0,0 +1,740 @@
...
+                            }
+                        }
+                    }
+                }
+            },
+            "dependsOn": [
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: move this above properties

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 15]
```diff
@@ -0,0 +1,740 @@
...
+                "[resourceId('Microsoft.Web/connections', parameters('Service Bus connection name'))]"
+            ]
+        },
+        {
+            "comments": "Generalized from resource: '/subscriptions/fdf3a3a3-c8f5-472f-8367-6a9a4a6c11a9/resourceGroups/CorrelatingMessages/providers/Microsoft.ServiceBus/namespaces/Router'.",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: remove comments

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 16]
```diff
@@ -0,0 +1,740 @@
...
+            "type": "Microsoft.ServiceBus/namespaces",
+            "sku": {
+                "name": "Standard",
+                "tier": "Standard"
+            },
+            "name": "[parameters('Service Bus Namespace (should be globally unique!)')]",
+            "apiVersion": "2017-04-01",
+            "location": "West Europe",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: use a parameter or resourceGroup().location

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 17]
```diff
@@ -0,0 +1,740 @@
...
+            "tags": {},
+            "scale": null,
+            "properties": {
+                "metricId": "[concat('fdf3a3a3-c8f5-472f-8367-6a9a4a6c11a9:router', parameters('Service Bus Namespace (should be globally unique!)'))]",
+                "createdAt": "2018-04-07T12:06:43.41Z",
+                "updatedAt": "2018-04-07T12:07:09.597Z",
+                "serviceBusEndpoint": "[concat('https://', parameters('Service Bus Namespace (should be globally unique!)'),'.servicebus.windows.net:443/')]",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: use reference() to get the uri

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 18]
```diff
@@ -0,0 +1,740 @@
...
+                "status": "Active"
+            },
+            "dependsOn": []
+        },
+        {
+            "comments": "Generalized from resource: '/subscriptions/fdf3a3a3-c8f5-472f-8367-6a9a4a6c11a9/resourceGroups/CorrelatingMessages/providers/Microsoft.Web/connections/servicebus'.",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: remove

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 19]
```diff
@@ -0,0 +1,740 @@
...
+            "type": "Microsoft.Web/connections",
+            "name": "[parameters('Service Bus connection name')]",
+            "apiVersion": "2016-06-01",
+            "location": "westeurope",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: use a parameter or resourceGroup().location

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 20]
```diff
@@ -0,0 +1,740 @@
...
+            "scale": null,
+            "properties": {
+                "displayName": "Router",
+                "customParameterValues": {},
+                "api": {
+                    "id": "[concat('/subscriptions/fdf3a3a3-c8f5-472f-8367-6a9a4a6c11a9/providers/Microsoft.Web/locations/westeurope/managedApis/', parameters('Service Bus connection name'))]"
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: use a valid resourceId()

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 21]
```diff
@@ -0,0 +1,740 @@
...
+                }
+            },
+            "dependsOn": []
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: remove

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 22]
```diff
@@ -0,0 +1,740 @@
...
+        },
+        {
+            "comments": "Generalized from resource: '/subscriptions/fdf3a3a3-c8f5-472f-8367-6a9a4a6c11a9/resourcegroups/CorrelatingMessages/providers/Microsoft.ServiceBus/namespaces/Router/AuthorizationRules/RootManageSharedAccessKey'.",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: remove

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 23]
```diff
@@ -0,0 +1,740 @@
...
+            "type": "Microsoft.ServiceBus/namespaces/AuthorizationRules",
+            "name": "[concat(parameters('Service Bus Namespace (should be globally unique!)'), '/', 'RootManageSharedAccessKey')]",
+            "apiVersion": "2017-04-01",
+            "location": "West Europe",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: use a parameter or resourceGroup().location

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 24]
```diff
@@ -0,0 +1,740 @@
...
+                    "Manage",
+                    "Send"
+                ]
+            },
+            "dependsOn": [
+                "[resourceId('Microsoft.ServiceBus/namespaces', parameters('Service Bus Namespace (should be globally unique!)'))]"
+            ]
+        },
+        {
+            "comments": "Generalized from resource: '/subscriptions/fdf3a3a3-c8f5-472f-8367-6a9a4a6c11a9/resourcegroups/CorrelatingMessages/providers/Microsoft.ServiceBus/namespaces/Router/topics/messagerouting'.",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: remove

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 25]
```diff
@@ -0,0 +1,740 @@
...
+            "type": "Microsoft.ServiceBus/namespaces/topics",
+            "name": "[concat(parameters('Service Bus Namespace (should be globally unique!)'), '/', parameters('Message routing Topic name'))]",
+            "apiVersion": "2017-04-01",
+            "location": "West Europe",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: use a parameter or resourceGroup().location

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 26]
```diff
@@ -0,0 +1,740 @@
...
+                "autoDeleteOnIdle": "P10675199DT2H48M5.4775807S",
+                "enablePartitioning": true,
+                "enableExpress": false
+            },
+            "dependsOn": [
+                "[resourceId('Microsoft.ServiceBus/namespaces', parameters('Service Bus Namespace (should be globally unique!)'))]"
+            ]
+        },
+        {
+            "comments": "Generalized from resource: '/subscriptions/fdf3a3a3-c8f5-472f-8367-6a9a4a6c11a9/resourcegroups/CorrelatingMessages/providers/Microsoft.ServiceBus/namespaces/Router/topics/messagerouting/subscriptions/ToBackendLogicApp'.",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: remove

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 27]
```diff
@@ -0,0 +1,740 @@
...
+            "type": "Microsoft.ServiceBus/namespaces/topics/subscriptions",
+            "name": "[concat(parameters('Service Bus Namespace (should be globally unique!)'), '/', parameters('Message routing Topic name'), '/', parameters('Backend Logic App subscription name'))]",
+            "apiVersion": "2017-04-01",
+            "location": "West Europe",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: use a parameter or resourceGroup().location

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 28]
```diff
@@ -0,0 +1,740 @@
...
+                "enableBatchedOperations": false,
+                "autoDeleteOnIdle": "P10675199DT2H48M5.4775807S"
+            },
+            "dependsOn": [
+                "[resourceId('Microsoft.ServiceBus/namespaces', parameters('Service Bus Namespace (should be globally unique!)'))]",
+                "[resourceId('Microsoft.ServiceBus/namespaces/topics', parameters('Service Bus Namespace (should be globally unique!)'), parameters('Message routing Topic name'))]"
+            ]
+        },
+        {
+            "comments": "Generalized from resource: '/subscriptions/fdf3a3a3-c8f5-472f-8367-6a9a4a6c11a9/resourcegroups/CorrelatingMessages/providers/Microsoft.ServiceBus/namespaces/Router/topics/messagerouting/subscriptions/ToClientLogicApp'.",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: remove

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 29]
```diff
@@ -0,0 +1,740 @@
...
+            "type": "Microsoft.ServiceBus/namespaces/topics/subscriptions",
+            "name": "[concat(parameters('Service Bus Namespace (should be globally unique!)'), '/', parameters('Message routing Topic name'), '/', parameters('Client Logic App subscription name'))]",
+            "apiVersion": "2017-04-01",
+            "location": "West Europe",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: use a parameter or resourceGroup().location

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 30]
```diff
@@ -0,0 +1,740 @@
...
+                "enableBatchedOperations": false,
+                "autoDeleteOnIdle": "P10675199DT2H48M5.4775807S"
+            },
+            "dependsOn": [
+                "[resourceId('Microsoft.ServiceBus/namespaces', parameters('Service Bus Namespace (should be globally unique!)'))]",
+                "[resourceId('Microsoft.ServiceBus/namespaces/topics', parameters('Service Bus Namespace (should be globally unique!)'), parameters('Message routing Topic name'))]"
+            ]
+        },
+        {
+            "comments": "Generalized from resource: '/subscriptions/fdf3a3a3-c8f5-472f-8367-6a9a4a6c11a9/resourcegroups/CorrelatingMessages/providers/Microsoft.ServiceBus/namespaces/Router/topics/messagerouting/subscriptions/ToTransformationLogicApp'.",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: remove

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 31]
```diff
@@ -0,0 +1,740 @@
...
+            "type": "Microsoft.ServiceBus/namespaces/topics/subscriptions",
+            "name": "[concat(parameters('Service Bus Namespace (should be globally unique!)'), '/', parameters('Message routing Topic name'), '/', parameters('Transformation Logic App subscription name'))]",
+            "apiVersion": "2017-04-01",
+            "location": "West Europe",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: use a parameter or resourceGroup().location

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 32]
```diff
@@ -0,0 +1,740 @@
...
+                "requiresSession": true,
+                "defaultMessageTimeToLive": "P14D",
+                "deadLetteringOnMessageExpiration": false,
+                "deadLetteringOnFilterEvaluationExceptions": false,
+                "maxDeliveryCount": 10,
+                "status": "Active",
+                "enableBatchedOperations": false,
+                "autoDeleteOnIdle": "P10675199DT2H48M5.4775807S"
+            },
+            "dependsOn": [
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: move this above properties

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 33]
```diff
@@ -0,0 +1,740 @@
...
+                "[resourceId('Microsoft.ServiceBus/namespaces', parameters('Service Bus Namespace (should be globally unique!)'))]",
+                "[resourceId('Microsoft.ServiceBus/namespaces/topics', parameters('Service Bus Namespace (should be globally unique!)'), parameters('Message routing Topic name'))]"
+            ]
+        },
+        {
+            "comments": "Generalized from resource: '/subscriptions/fdf3a3a3-c8f5-472f-8367-6a9a4a6c11a9/resourcegroups/CorrelatingMessages/providers/Microsoft.ServiceBus/namespaces/Router/topics/messagerouting/subscriptions/ToBackendLogicApp/rules/$Default'.",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: remove

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 34]
```diff
@@ -0,0 +1,740 @@
...
+            "type": "Microsoft.ServiceBus/namespaces/topics/subscriptions/rules",
+            "name": "[concat(parameters('Service Bus Namespace (should be globally unique!)'), '/', parameters('Message routing Topic name'), '/', parameters('Backend Logic App subscription name'), '/', 'RouteToBackendLogicApp')]",
+            "apiVersion": "2017-04-01",
+            "location": "West Europe",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: use a parameter or resourceGroup().location

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 35]
```diff
@@ -0,0 +1,740 @@
...
+            "scale": null,
+            "properties": {
+                "action": {},
+                "filterType": "SqlFilter",
+                "sqlFilter": {
+                    "sqlExpression": "Source='TransformationLogicApp'"
+                }
+            },
+            "dependsOn": [
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: move this above properties

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 36]
```diff
@@ -0,0 +1,740 @@
...
+                "[resourceId('Microsoft.ServiceBus/namespaces', parameters('Service Bus Namespace (should be globally unique!)'))]",
+                "[resourceId('Microsoft.ServiceBus/namespaces/topics', parameters('Service Bus Namespace (should be globally unique!)'), parameters('Message routing Topic name'))]",
+                "[resourceId('Microsoft.ServiceBus/namespaces/topics/subscriptions', parameters('Service Bus Namespace (should be globally unique!)'), parameters('Message routing Topic name'), parameters('Backend Logic App subscription name'))]"
+            ]
+        },
+        {
+            "comments": "Generalized from resource: '/subscriptions/fdf3a3a3-c8f5-472f-8367-6a9a4a6c11a9/resourcegroups/CorrelatingMessages/providers/Microsoft.ServiceBus/namespaces/Router/topics/messagerouting/subscriptions/ToClientLogicApp/rules/$Default'.",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: remove

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 37]
```diff
@@ -0,0 +1,740 @@
...
+            "type": "Microsoft.ServiceBus/namespaces/topics/subscriptions/rules",
+            "name": "[concat(parameters('Service Bus Namespace (should be globally unique!)'), '/', parameters('Message routing Topic name'), '/', parameters('Client Logic App subscription name'), '/', 'RouteToClientLogicApp')]",
+            "apiVersion": "2017-04-01",
+            "location": "West Europe",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: use a parameter or resourceGroup().location

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 38]
```diff
@@ -0,0 +1,740 @@
...
+            "scale": null,
+            "properties": {
+                "action": {},
+                "filterType": "SqlFilter",
+                "sqlFilter": {
+                    "sqlExpression": "Source='BackendLogicApp'"
+                }
+            },
+            "dependsOn": [
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: move this above properties

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 39]
```diff
@@ -0,0 +1,740 @@
...
+                "[resourceId('Microsoft.ServiceBus/namespaces', parameters('Service Bus Namespace (should be globally unique!)'))]",
+                "[resourceId('Microsoft.ServiceBus/namespaces/topics', parameters('Service Bus Namespace (should be globally unique!)'), parameters('Message routing Topic name'))]",
+                "[resourceId('Microsoft.ServiceBus/namespaces/topics/subscriptions', parameters('Service Bus Namespace (should be globally unique!)'), parameters('Message routing Topic name'), parameters('Client Logic App subscription name'))]"
+            ]
+        },
+        {
+            "comments": "Generalized from resource: '/subscriptions/fdf3a3a3-c8f5-472f-8367-6a9a4a6c11a9/resourcegroups/CorrelatingMessages/providers/Microsoft.ServiceBus/namespaces/Router/topics/messagerouting/subscriptions/ToTransformationLogicApp/rules/$Default'.",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: remove

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 40]
```diff
@@ -0,0 +1,740 @@
...
+            "type": "Microsoft.ServiceBus/namespaces/topics/subscriptions/rules",
+            "name": "[concat(parameters('Service Bus Namespace (should be globally unique!)'), '/', parameters('Message routing Topic name'), '/', parameters('Transformation Logic App subscription name'), '/', 'RouteToTransformationLogicApp')]",
+            "apiVersion": "2017-04-01",
+            "location": "West Europe",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: use a parameter or resourceGroup().location

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 41]
```diff
@@ -0,0 +1,740 @@
...
+            "scale": null,
+            "properties": {
+                "action": {},
+                "filterType": "SqlFilter",
+                "sqlFilter": {
+                    "sqlExpression": "Source='ClientLogicApp'"
+                }
+            },
+            "dependsOn": [
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: move this above properties

**Comment by EldertGrootenboer** [comment]
Requested changes have been implemented.

File: 201-logic-app-correlation-using-servicebus/azuredeploy.json [hunk 42]
```diff
@@ -517,80 +517,80 @@
...
+            "location": "[resourceGroup().location]",
             "tags": {},
             "scale": null,
             "properties": {
-                "metricId": "[concat('fdf3a3a3-c8f5-472f-8367-6a9a4a6c11a9:router', parameters('Service Bus Namespace (should be globally unique!)'))]",
+                "metricId": "[concat(subscription().subscriptionId, ':router', parameters('serviceBusNamespace'))]",
                 "createdAt": "2018-04-07T12:06:43.41Z",
                 "updatedAt": "2018-04-07T12:07:09.597Z",
-                "serviceBusEndpoint": "[concat('https://', parameters('Service Bus Namespace (should be globally unique!)'),'.servicebus.windows.net:443/')]",
+                "serviceBusEndpoint": "[concat('https://', parameters('serviceBusNamespace'),'.servicebus.windows.net:443/')]",
```
Comment (bmoore-msft) [inline 201-logic-app-correlation-using-servicebus/azuredeploy.json]: use reference() to get the endpoint

**Pull request merged**